CMakeLists.txt

```diff
@@ -155,7 +155,7 @@ if(CC_TARGETS_AMD64)
     add_definitions(-D_M_X64_OR_ARM64)
     add_compile_options(-msse4.2)
 
-    set(CAN_BUILD_WABT 1)
+    #set(CAN_BUILD_WABT 1)
```

Comment:
Why this change? (cc @Cellule)

Reply:
Indeed, we need wabt for some of our test on xplat (agreed very few passes).
What we need to do is not set this on release builds, but leave it on test/debug builds

Reply:
no worries! was experimenting with the CI. libwabt is no longer part of the binary for xplat release builds